[2025-02-02T13:22:49.650+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T13:22:49.672+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T13:22:49.684+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T13:22:49.686+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-02T13:22:49.702+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): spark_consumer> on 2025-02-01 00:00:00+00:00
[2025-02-02T13:22:49.713+0000] {standard_task_runner.py:72} INFO - Started process 330 to run task
[2025-02-02T13:22:49.718+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'save_to_minio', 'spark_consumer', 'scheduled__2025-02-01T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpmmbajj6c']
[2025-02-02T13:22:49.722+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask spark_consumer
[2025-02-02T13:22:49.778+0000] {task_command.py:467} INFO - Running <TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [running]> on host e6ed12b0dfa5
[2025-02-02T13:22:49.860+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andrewsis' AIRFLOW_CTX_DAG_ID='save_to_minio' AIRFLOW_CTX_TASK_ID='spark_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-01T00:00:00+00:00'
[2025-02-02T13:22:49.862+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T13:22:55.020+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/main_pipeline.py", line 63, in spark_consumer
    .getOrCreate()
     ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/sql/session.py", line 497, in getOrCreate
    sc = SparkContext.getOrCreate(sparkConf)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/context.py", line 515, in getOrCreate
    SparkContext(conf=conf or SparkConf())
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/context.py", line 201, in __init__
    SparkContext._ensure_initialized(self, gateway=gateway, conf=conf)
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/context.py", line 436, in _ensure_initialized
    SparkContext._gateway = gateway or launch_gateway(conf)
                                       ^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/java_gateway.py", line 107, in launch_gateway
    raise PySparkRuntimeError(
pyspark.errors.exceptions.base.PySparkRuntimeError: [JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.
[2025-02-02T13:22:55.043+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=save_to_minio, task_id=spark_consumer, run_id=scheduled__2025-02-01T00:00:00+00:00, execution_date=20250201T000000, start_date=20250202T132249, end_date=20250202T132255
[2025-02-02T13:22:55.068+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T13:22:55.070+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 20 for task spark_consumer ([JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.; 330)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/main_pipeline.py", line 63, in spark_consumer
    .getOrCreate()
     ^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/sql/session.py", line 497, in getOrCreate
    sc = SparkContext.getOrCreate(sparkConf)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/context.py", line 515, in getOrCreate
    SparkContext(conf=conf or SparkConf())
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/context.py", line 201, in __init__
    SparkContext._ensure_initialized(self, gateway=gateway, conf=conf)
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/context.py", line 436, in _ensure_initialized
    SparkContext._gateway = gateway or launch_gateway(conf)
                                       ^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pyspark/java_gateway.py", line 107, in launch_gateway
    raise PySparkRuntimeError(
pyspark.errors.exceptions.base.PySparkRuntimeError: [JAVA_GATEWAY_EXITED] Java gateway process exited before sending its port number.
[2025-02-02T13:22:55.109+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-02-02T13:22:55.134+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T13:22:55.137+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-02T13:38:58.236+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T13:38:58.248+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T13:38:58.254+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T13:38:58.255+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-02T13:38:58.265+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): spark_consumer> on 2025-02-01 00:00:00+00:00
[2025-02-02T13:38:58.269+0000] {standard_task_runner.py:72} INFO - Started process 1460 to run task
[2025-02-02T13:38:58.271+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'save_to_minio', 'spark_consumer', 'scheduled__2025-02-01T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpsyi0cy2n']
[2025-02-02T13:38:58.273+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask spark_consumer
[2025-02-02T13:38:58.306+0000] {task_command.py:467} INFO - Running <TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [running]> on host 64f2a5764477
[2025-02-02T13:38:58.369+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andrewsis' AIRFLOW_CTX_DAG_ID='save_to_minio' AIRFLOW_CTX_TASK_ID='spark_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-01T00:00:00+00:00'
[2025-02-02T13:38:58.370+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T13:39:09.018+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2025-02-02T13:39:09.788+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:10.537+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:11.281+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:12.360+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:13.148+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:13.948+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:14.752+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:15.667+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:16.497+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:17.276+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:18.385+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:19.194+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:20.005+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:20.748+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:21.501+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:22.250+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:23.003+0000] {sql.py:681} INFO - Loaded 1000 rows into crypto_data so far
[2025-02-02T13:39:23.393+0000] {sql.py:681} INFO - Loaded 520 rows into crypto_data so far
[2025-02-02T13:39:23.394+0000] {sql.py:694} INFO - Done loading. Loaded a total of 17520 rows into crypto_data
[2025-02-02T13:39:23.402+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T13:39:23.409+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T13:39:23.410+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=save_to_minio, task_id=spark_consumer, run_id=scheduled__2025-02-01T00:00:00+00:00, execution_date=20250201T000000, start_date=20250202T133858, end_date=20250202T133923
[2025-02-02T13:39:23.440+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T13:39:23.452+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T13:39:23.454+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-02T18:59:25.043+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T18:59:25.063+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T18:59:25.075+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T18:59:25.077+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-02T18:59:25.091+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): spark_consumer> on 2025-02-01 00:00:00+00:00
[2025-02-02T18:59:25.096+0000] {standard_task_runner.py:72} INFO - Started process 3740 to run task
[2025-02-02T18:59:25.099+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'save_to_minio', 'spark_consumer', 'scheduled__2025-02-01T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmp7ya09oql']
[2025-02-02T18:59:25.101+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask spark_consumer
[2025-02-02T18:59:25.141+0000] {task_command.py:467} INFO - Running <TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [running]> on host 15904aeddad7
[2025-02-02T18:59:25.209+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andrewsis' AIRFLOW_CTX_DAG_ID='save_to_minio' AIRFLOW_CTX_TASK_ID='spark_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-01T00:00:00+00:00'
[2025-02-02T18:59:25.211+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T18:59:32.412+0000] {java_gateway.py:2273} INFO - Callback Server Starting
[2025-02-02T18:59:32.414+0000] {java_gateway.py:2275} INFO - Socket listening on ('127.0.0.1', 42823)
[2025-02-02T18:59:34.509+0000] {clientserver.py:561} INFO - Python Server ready to receive messages
[2025-02-02T18:59:34.510+0000] {clientserver.py:575} INFO - Received command c on object id p0
[2025-02-02T18:59:34.513+0000] {main_pipeline.py:161} INFO - Writing batch 0 to PostgreSQL.
[2025-02-02T18:59:34.603+0000] {main_pipeline.py:162} INFO - Writing batch with data: None to PostgreSQL.
[2025-02-02T18:59:34.907+0000] {main_pipeline.py:164} INFO - Batch 0 successfully written to PostgreSQL.
[2025-02-02T18:59:52.745+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T18:59:52.755+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T18:59:52.757+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=save_to_minio, task_id=spark_consumer, run_id=scheduled__2025-02-01T00:00:00+00:00, execution_date=20250201T000000, start_date=20250202T185925, end_date=20250202T185952
[2025-02-02T18:59:52.787+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T18:59:52.798+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T18:59:52.800+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-02T19:22:56.767+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T19:22:56.778+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T19:22:56.784+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [queued]>
[2025-02-02T19:22:56.785+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-02T19:22:56.793+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): spark_consumer> on 2025-02-01 00:00:00+00:00
[2025-02-02T19:22:56.797+0000] {standard_task_runner.py:72} INFO - Started process 1899 to run task
[2025-02-02T19:22:56.799+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'save_to_minio', 'spark_consumer', 'scheduled__2025-02-01T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmp3qdg50rg']
[2025-02-02T19:22:56.801+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask spark_consumer
[2025-02-02T19:22:56.832+0000] {task_command.py:467} INFO - Running <TaskInstance: save_to_minio.spark_consumer scheduled__2025-02-01T00:00:00+00:00 [running]> on host dfbcbb9a3a68
[2025-02-02T19:22:56.886+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andrewsis' AIRFLOW_CTX_DAG_ID='save_to_minio' AIRFLOW_CTX_TASK_ID='spark_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-01T00:00:00+00:00'
[2025-02-02T19:22:56.887+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T19:23:02.759+0000] {main_pipeline.py:73} INFO - DataFrame является потоком
[2025-02-02T19:23:03.293+0000] {main_pipeline.py:149} INFO - df_parsed является потоком
[2025-02-02T19:23:03.311+0000] {java_gateway.py:2273} INFO - Callback Server Starting
[2025-02-02T19:23:03.315+0000] {java_gateway.py:2275} INFO - Socket listening on ('127.0.0.1', 34853)
[2025-02-02T19:23:07.361+0000] {clientserver.py:561} INFO - Python Server ready to receive messages
[2025-02-02T19:23:07.362+0000] {clientserver.py:575} INFO - Received command c on object id p0
[2025-02-02T19:23:08.292+0000] {main_pipeline.py:175} INFO - Запись пакета 0 с данными: Row(symbol='ETHBTC', price_change=Decimal('-0.00165000'), price_change_percent=Decimal('-5.22000000'), weighted_avg_price=Decimal('0.03083062'), prev_close_price=Decimal('0.03160000'), last_price=Decimal('0.02996000'), last_qty=Decimal('3.43440000'), bid_price=Decimal('0.02996000'), bid_qty=Decimal('25.82290000'), ask_price=Decimal('0.02997000'), ask_qty=Decimal('21.77020000'), open_price=Decimal('0.03161000'), high_price=Decimal('0.03164000'), low_price=Decimal('0.02983000'), volume=Decimal('37675.78250000'), quote_volume=Decimal('1161.56779859'), close_time=datetime.datetime(2025, 2, 2, 19, 19, 50), first_id=485441387, last_id=485575574, count=134188, trend=False) в PostgreSQL.
[2025-02-02T19:23:08.294+0000] {main_pipeline.py:178} INFO - Writing batch 0 to PostgreSQL.
[2025-02-02T19:23:08.348+0000] {main_pipeline.py:179} INFO - Writing batch with data: Row(symbol='ETHBTC', price_change=Decimal('-0.00165000'), price_change_percent=Decimal('-5.22000000'), weighted_avg_price=Decimal('0.03083062'), prev_close_price=Decimal('0.03160000'), last_price=Decimal('0.02996000'), last_qty=Decimal('3.43440000'), bid_price=Decimal('0.02996000'), bid_qty=Decimal('25.82290000'), ask_price=Decimal('0.02997000'), ask_qty=Decimal('21.77020000'), open_price=Decimal('0.03161000'), high_price=Decimal('0.03164000'), low_price=Decimal('0.02983000'), volume=Decimal('37675.78250000'), quote_volume=Decimal('1161.56779859'), close_time=datetime.datetime(2025, 2, 2, 19, 19, 50), first_id=485441387, last_id=485575574, count=134188, trend=False) to PostgreSQL.
[2025-02-02T19:23:11.207+0000] {main_pipeline.py:181} INFO - Batch 0 successfully written to PostgreSQL.
[2025-02-02T19:23:23.851+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T19:23:23.861+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T19:23:23.862+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=save_to_minio, task_id=spark_consumer, run_id=scheduled__2025-02-01T00:00:00+00:00, execution_date=20250201T000000, start_date=20250202T192256, end_date=20250202T192323
[2025-02-02T19:23:23.885+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T19:23:23.897+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T19:23:23.900+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
